src/Compilers/CSharp/Test/Semantic/Semantics/InheritanceBindingTests.cs

```diff
@@ -4771,11 +4771,20 @@ public class Derived : Base<int>, Interface<int, int>
 {
 }
 ";
-            CompileAndVerifyDiagnostics(text, new ErrorDescription[] {
-                new ErrorDescription { Code = (int)ErrorCode.WRN_MultipleRuntimeImplementationMatches, Line = 15, Column = 35, IsWarning = true }, //Both Base methods implement Interface.Method(int)
-                new ErrorDescription { Code = (int)ErrorCode.WRN_MultipleRuntimeImplementationMatches, Line = 15, Column = 35, IsWarning = true }, //Both Base methods implement Interface.Method(T)
-                new ErrorDescription { Code = (int)ErrorCode.WRN_MultipleRuntimeImplementationMatches, Line = 15, Column = 35, IsWarning = true }, //Both Base methods implement Interface.Method(U)
-            });
+            //Both Base methods implement Interface.Method(int)
+            //Both Base methods implement Interface.Method(T)
+            //Both Base methods implement Interface.Method(U)
+            CreateCompilation(text).VerifyDiagnostics(
```

Comment:
>VerifyDiagnostics [](start = 36, length = 17)

Is this test going to break once we start testing against new core CLR? Consider adding a comment with a hint to split the test to explicitly target Core CLR and other frameworks once that happens.

Reply:
No, the changes in this PR have no effect on this test at compile-time or runtime. I just used newer APIs in the test to make it easier to understand.

---
In reply to: [446344222](https://github.com/dotnet/roslyn/pull/44025#discussion_r446344222) [](ancestors = 446344222)

Reply:
> No, the changes in this PR have no effect on this test at compile-time or runtime. I just used newer APIs in the test to make it easier to understand.

Are you saying that we will keep reporting the warnings even when the target runtime supports covariant returns?

---
In reply to: [446406777](https://github.com/dotnet/roslyn/pull/44025#discussion_r446406777) [](ancestors = 446406777,446344222)

Reply:
Yes, these diagnostics have nothing much to do with the changes in this PR. These are ambiguities in implicit interface implementations, which we are not touching.

---
In reply to: [446407986](https://github.com/dotnet/roslyn/pull/44025#discussion_r446407986) [](ancestors = 446407986,446406777,446344222)